pages/advocates/index.vue

```diff
@@ -91,6 +72,10 @@ import MapSection from '~/components/advocates/MapSection.vue'
 import AdvocateCard from '~/components/advocates/AdvocateCard.vue'
 import CompactFeature from '~/components/ui/CompactFeature.vue'
 
+type Benefit = {
+  [key: string]: string
```

Comment:
Since we know the shape of the `Benefit`, it would be better to have it fully typed. I mean:
```ts
type Benefit = {
  'icon': string,
  ...
}
```

However, trying not to repeat ourselves too much, we can use:

```ts
type Benefit = Pick<CompactFeature, 'id'|'title'|'description'>
```